docs/user-manuals/fine-grained-cpu-orchestration.md

```diff
@@ -0,0 +1,138 @@
+# Fine-grained CPU Orchestration
+
+Fine-grained CPU Orchestration is an ability of koord-scheduler for improving the performance of CPU-sensitive workloads.
+
+## Introduction
+
+There is an increasing number of systems that leverage a combination of CPUs and hardware accelerators to support
+latency-critical execution and high-throughput parallel computation. A high-performance environment is expected in
+plenty of applications including in telecommunications, scientific computing, machine learning, financial services, and
+data analytics.
+
+However, pods in the Kubernetes cluster may interfere with others' running when they share the same physical resources
+and both demand many resources. The sharing of CPU resources is almost inevitable. e.g. SMT threads (i.e. logical
+processors) share execution units of the same core, and cores in the same chip share one last-level cache. The resource
+contention can slow down the running of these CPU-sensitive workloads, resulting in high response latency (RT).
+
+To improve the performance of CPU-sensitive workloads, koord-scheduler provides a mechanism of fine-grained CPU
+orchestration. It enhances the CPU management of Kubernetes and supports detailed NUMA-locality and CPU exclusions.
+
+For more information, please see [Design: Fine-grained CPU orchestration](/docs/designs/fine-grained-cpu-orchestration).
+
+## Setup
+
+### Prerequisite
+
+- Kubernetes >= 1.18
+- Koordinator >= 0.5
+
+### Installation
+
+Please make sure Koordinator components are correctly installed in your cluster. If not, please refer to [Installation](/docs/installation).
+
+### Configurations
+
+You can enable load-aware scheduling and configure the rules by modifying the ConfigMap `koord-scheduler-config` in the helm chart.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: koord-scheduler-config
+  ...
+data:
+  koord-scheduler-config: |
+    apiVersion: kubescheduler.config.k8s.io/v1beta2
+    kind: KubeSchedulerConfiguration
+    profiles:
+      - schedulerName: koord-scheduler
+      - pluginConfig:
+        - name: NodeNUMAResource
+          args:
+            apiVersion: kubescheduler.config.k8s.io/v1beta2
+            kind: NodeNUMAResource
+            # the CPU binding policy ('None', 'FullPCPUs', 'SpreadByPCPUs', 'ConstrainedBurst')
+            preferredCPUBindPolicy: FullPCPUs
+            # the NUMA allocation strategy ('MostAllocated', 'LeastAllocated', 'DistributeEvenly')
+            numaAllocateStrategy: MostAllocated
+            # the scoring strategy
+            scoringStrategy:
+              # the kind of scoring strategy ('MostAllocated', 'BalancedAllocation', 'LeastAllocated')
```

Comment:
BalancedAllocation is not currently supported.

Reply:
fixed